chisel.lua

```diff
@@ -27,18 +27,14 @@ jonez.chisel.register_chiselable_stair_and_slab = function(node_subname, group_s
 end
 
 local function chisel_interact(player, pointed_thing, is_right_click)
+	if not player then return end
 	if pointed_thing.type ~= "node" then
 		return
 	end
 
 	local pos = pointed_thing.under
-	local is_sneak = player and player:get_player_control().sneak or false
-	local player_name = player and player:get_player_name()
-
-	-- A true player is required
-	if not player_name then
-		return
-	end
+	local is_sneak = player:get_player_control().sneak or false
```

Comment:
The `or false` is redundent.
```suggestion
	local is_sneak = player:get_player_control().sneak
```